internal/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
 // +build tools
```

Comment:
Should we leave only one build tag?

Reply:
I agree. Should we go by what the linter suggests (`go:build tools`)?

Reply:
It’s for compatibility reasons to older go versions. I’d say we keep them for now.

Reply:
Ack!